tools/SourceKit/docs/Protocol.md

```diff
@@ -40,8 +40,8 @@ provided.
 
 | Request Name | Request Key | Description |
 | -------------:|:------------|:------------|
-| `codecomplete` | `codecomplete` |  Given a file will open a code-completion session which can be filtered upon using `codecomplete.update`. Each session must be closed using `codecomplete.close`.|
-| `open` | `codecomplete.open` | Open a code-completion session for the given input file and offset, and return the initial list of completions. |
+| `codecomplete` | `codecomplete` | Open a code-completion session for the given input file and offset, and return the initial list of completions. |
```

Comment:
Abstractly you could sort of think of it that way, but it doesn't really work that way.  The "session" for the  requests open/close/update is really a way of storing the results we get back from code-completion in the compiler and manipulating them.  Code-complete just returns the results and is done.